Games/Chess/ChessWidget.cpp

```diff
@@ -27,17 +27,20 @@
 #include "ChessWidget.h"
 #include "PromotionDialog.h"
 #include <AK/String.h>
+#include <LibCore/DateTime.h>
+#include <LibCore/File.h>
 #include <LibGUI/MessageBox.h>
 #include <LibGUI/Painter.h>
 #include <LibGfx/Font.h>
+#include <unistd.h>
 
 ChessWidget::ChessWidget(const StringView& set)
 {
     set_piece_set(set);
 }
 
 ChessWidget::ChessWidget()
-    : ChessWidget("test")
+    : ChessWidget("stelar7")
```

Comment:
I take offense to this :^)

Reply:
You're welcome to override the default in your `Chess.ini` configuration file :P

Reply:
I don't know the backstory here... I was just thinking from a first time user's perspective.
I can revert the change  : )

Reply:
just kidding around. it's probably for the best long term